news/2024/2024-10-28-performance-points-star-rating-updates.md

```diff
@@ -163,7 +165,7 @@ A [change](https://github.com/ppy/osu/pull/20963) proposed by [Natelytle](https:
 
 The current way accuracy scaling works in osu!taiko is very simple. It takes the decimal accuracy of the score, raises it to the power of 15, and multiplies the result by a constant determined by the overall difficulty. The problem with this is that if you keep the spread of your hit timings (also known as UR) the same, your accuracy does not decrease at a constant rate as you increase the overall difficulty.
 
-The consequence of this is that if you take the same score and increase the overall difficulty on it, eventually you will reach a point where the accuracy drops off faster than the constant bonus for OD increases. This means the exact same score can *lose* pp if the overall difficulty is too high.
+The consequence of this is that if you take the same score and increase the overall difficulty on it, eventually you will reach a point where the accuracy drops off faster than the constant bonus for OD increases. This means the exact same score can *lose* pp if the overall difficulty is too high. The Easy mod also is given a significantly harsher penalty.
```

Comment:
Easy mod seems completely unrelated to the rest of the paragraph if I'm understanding it right

Reply:
fixed.